server/middleware/setUpHealthChecks.ts

```diff
@@ -1,41 +1,23 @@
 import express, { type Router } from 'express'
+import { monitoringMiddleware, endpointHealthComponent } from '@ministryofjustice/hmpps-monitoring'
 
 import type { ApplicationInfo } from '../applicationInfo'
+import logger from '../../logger'
 import config from '../config'
-import healthcheck from '../services/healthCheck'
 
 export default function setUpHealthChecks(applicationInfo: ApplicationInfo): Router {
   const router = express.Router()
 
-  router.get('/health', (req, res, next): void => {
-    healthcheck(applicationInfo, result => {
-      if (result.status !== 'UP') {
-        res.status(503)
-      }
-      res.json(result)
-    })
-  })
+  const apiConfig = Object.entries(config.apis)
 
-  router.get('/ping', (req, res): void => {
-    res.send({
-      status: 'UP',
-    })
+  const middleware = monitoringMiddleware({
+    applicationInfo,
+    healthComponents: apiConfig.map(([name, options]) => endpointHealthComponent(logger, name, options)),
   })
 
-  router.get('/info', (req, res): void => {
-    res.json({
-      git: {
-        branch: applicationInfo.branchName,
-      },
-      build: {
-        artifact: applicationInfo.applicationName,
-        version: applicationInfo.buildNumber,
-        name: applicationInfo.applicationName,
-      },
-      productId: applicationInfo.productId,
-      activeAgencies: config.activePrisons,
```

Comment:
this i supposed to drive the display of this service on DPS so it should not be removed

Reply:
I've reverted the commit that removed that test and added the `activeAgencies` field in the `ApplicationInfo` object...my first attempt wasn't successful as [monitoring has a mechanism to expose extra fields](https://github.com/ministryofjustice/hmpps-incident-reporting/pull/436/commits/12cdc8b29a7d6c2a7d65c9934e7a99197794e371).